dll/win32/uxtheme/lang/tr-TR.rc

```diff
@@ -0,0 +1,19 @@
+/*
+ * PROJECT:     ReactOS uxtheme.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATORS: Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
+ */
+
+LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
+
+/* Strings */
+
+STRINGTABLE
+BEGIN
+    IDS_MESSAGEBOX "İleti Kutusu"
```

Comment:
Does not match desk.cpl `IDS_MESSBOX "İleti Penceresi"`

Reply:
> Does not match desk.cpl `IDS_MESSBOX "İleti Penceresi"`

I will also change desk translation because the translations there are also wrong

Reply:
Ah alright, then please be sure to attach a screenshot that can confirm this in Windows with Turkish locale.

Reply:
![image](https://github.com/user-attachments/assets/d8c3456b-7c8b-4a50-88b2-a24f0c880fe9)

its now fully matches the desk.cpl translations

"İleti Kutusu" "Pasif Pencere"

Reply:
Thanks, got confirmation screenshot in #7308.